### PR TITLE
Add username field in user registration page

### PR DESCRIPTION
**Pull request by wlteo**
Added username field for user registration page

### CODE REVIEW

File: app/views/devise/registrations/new.html.erb [hunk 1]
```diff
@@ -4,6 +4,11 @@
   <%= devise_error_messages! %>
 
   <div class="field">
+    <%= f.label :username %><br />
+    <%= f.email_field :username, autofocus: true %>
```
Comment (sabrina-s) [inline app/views/devise/registrations/new.html.erb]: Should be f.text_field!

File: app/views/devise/registrations/new.html.erb [hunk 2]
```diff
@@ -4,6 +4,11 @@
...
+  </div>
+
+  <div class="field">
     <%= f.label :email %><br />
     <%= f.email_field :email, autofocus: true %>
```
Comment (sabrina-s) [inline app/views/devise/registrations/new.html.erb]: I think we can remove the autofocus here 😃

**Review by sabrina-s — CHANGES_REQUESTED** [review]

File: app/views/devise/registrations/new.html.erb (hunk 2 above)
Comment (wlteo) [reply app/views/devise/registrations/new.html.erb]: Done!

**Review by sabrina-s — APPROVED** [review]
LGTM! 👍

**Pull request merged**